### DISCUSSION TITLE
Can't type vietnamese?

### DISCUSSION THREAD

**Discussion by vantu-qvtech** [AvaloniaUI/Avalonia#11186] [Q&A]
Hi All 
I can't type vietnamese .I use  control  textbox avalonia 11..7
Ex : Đ, ă..etc 

thanks

**Comment by Gillibald** [comment]
So Vietnamese IME isn't working for you?

**Reply by vantu-qvtech** [reply]
yes

**Comment by uno-studio** [comment]
I also had the same problem after updating to v11. for 0.10.x everything is working.
In Vietnam, people use third party percussion, for example: UniKey, EvKey...
after updating to v11. Can't type Vietnamese in Avalonia anymore
